plugin/web/extensions/sidebar/utils/fieldGroups.ts

```diff
@@ -1,6 +1,6 @@
 import { Group } from "../core/types";
 
-export const fieldGroupMax = 10; // Lowering this from 10 will break some saved Data. Change only with utmost caution.
+export const fieldGroupMax = 24; // Lowering this from 10 will break some saved Data. Change only with utmost caution.
```

Comment:
Sorry can you update the comment too from 10 to 24?

Reply:
Done!